### DISCUSSION TITLE
Can we have async loading batch by batch.

### DISCUSSION THREAD

**Discussion by EVNIKHIL** [JedWatson/react-select#5442] [Ideas]
Suppose I have 100 options and don't want load the whole. as user scrolling to bottom can we load them async say 10 each in loading

**Comment by manjushsh** [comment]
Hi @EVNIKHIL,
You can try [react-select-async-paginate](https://www.npmjs.com/package/react-select-async-paginate)
Example: [Select](https://codesandbox.io/s/o75rno2w65?file=/src/index.js)
